app/components/Auction.js

```diff
@@ -167,6 +168,15 @@ export default class Auction extends Component {
     else return colors.blue;
   }
 
+  @computed
+  get endDate() {
```

Comment:
Hmm, the `Countdown` would get the auction do this calculation itself to not duplicate the logic. But now it's fine.

Reply:
But thoughts on this?

Reply:
Ideally would like to create a helper method that could be shared between the two components.  Countdown doesn't really need to know about an auction. Just needs to know about times.